crates/release_plz/src/args/mod.rs

```diff
@@ -49,9 +48,28 @@ const HELP_STYLES: Styles = Styles::styled()
 pub struct CliArgs {
     #[command(subcommand)]
     pub command: Command,
+    /// Print source location and additional information in logs.
+    /// The default log level is INFO.
```

Comment:
In order to use the first line as the short help text (`-h`), and reserve the rest for the long text (`--help`) add a blank line:

```suggestion
    /// Print source location and additional information in logs.
    ///
    /// The default log level is INFO.
```